src/main/java/org/folio/circulation/support/http/client/Limit.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.circulation.support.http.client;
+
+import static java.lang.Integer.MAX_VALUE;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.ext.web.client.HttpRequest;
+
+public class Limit implements QueryParameter {
+  private final Integer value;
+
+  public static Limit limit(int limit) {
+    return new Limit(limit);
+  }
+
+  public static Limit maximumLimit() {
+    return limit(MAX_VALUE);
+  }
+
+  public static Limit noLimit() {
+    return new Limit(null);
```

Comment:
Is it possible to create a single static instance and return it every time instead of constructing new instance?